### PR TITLE
calibre: 5.13.0 -> 5.16.1

### PR DESCRIPTION
**Pull request by eduardosm**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by r-rmcgibbo** [comment]
Result of `nixpkgs-review pr 119858` at c3d357bc run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built successfully:</summary>
  <ul>
    <li>calibre</li>
  </ul>
</details>
<details>
  <summary>1 suggestion:</summary>
  <ul>
    <li>warning: <a href="https://github.com/jtojnar/nixpkgs-hammering/blob/master/explanations/unnecessary-parallel-building.md">unnecessary-parallel-building</a>

Meson, CMake and qmake already set `enableParallelBuilding = true` by default so it is not necessary.

Near pkgs/applications/misc/calibre/default.nix:65:3:
```
   |
65 |   enableParallelBuilding = true;
   |   ^
```
</li>
  </ul>
</details>

**Comment by SuperSandro2000** [comment]
@eduardosm can you remove the not required enableParallelBuilding?

**Comment by eduardosm** [comment]
> @eduardosm can you remove the not required enableParallelBuilding?

Done

File: pkgs/applications/misc/calibre/default.nix [hunk 1]
```diff
@@ -26,11 +26,11 @@
 
 mkDerivation rec {
   pname = "calibre";
-  version = "5.13.0";
+  version = "5.16.1";
 
   src = fetchurl {
     url = "https://download.calibre-ebook.com/${version}/${pname}-${version}.tar.xz";
-    sha256 = "sha256-GDFAZxZmkio7e7kVjhYqhNdhXIlUPJF0iMWVl0uWVCM=";
+    sha256 = "1cbjrmrdxn3l65d60gi9bk53hr946vbkp1n6g6ryfd068ddw4dcm";
```
Comment (AndersonTorres) [inline pkgs/applications/misc/calibre/default.nix:33]:
```suggestion
    hash = "sha256-lTXCW0MGNOezecaGO9c2JGU4ylwpPmBaMXTY3nLNcrE=";
```

**Review by AndersonTorres — APPROVED** [review]

**Review by AndersonTorres — CHANGES_REQUESTED** [review]
Change the hash to the new SRI format, as indicated

File: pkgs/applications/misc/calibre/default.nix (hunk 1 above)
Comment (eduardosm) [reply pkgs/applications/misc/calibre/default.nix]: Done

**Review by AndersonTorres — APPROVED** [review]

**Pull request merged**